src/bar_chart.ts

```diff
@@ -24,6 +24,15 @@ class BarChart implements IAppView {
 
   private openHistogram2D;
 
+  private index = [];
+
+  private leftValue = [];
+
+  private widthBar = 15;
+
+  private w = 80;
```

Comment:
What is the width for? Because you have multiple width variables flying around. Can you please give it a more self-explanatory name?